apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-step/index.tsx

```diff
@@ -2,21 +2,19 @@
  * External dependencies
  */
 import * as React from 'react';
-import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import './styles.scss';
 
 export interface Props {
-	className?: string;
 	onPrevStep?: () => void;
 	onNextStep?: () => void;
 }
 
-const LaunchStep: React.FunctionComponent< Props > = ( { className, children } ) => {
-	return <div className={ classnames( 'nux-launch-step', className ) }>{ children }</div>;
+const LaunchStep: React.FunctionComponent< Props > = ( { children } ) => {
+	return <>{ children }</>;
```

Comment:
The goal is to have a flattened div to easily have scrollbars with sticky header & footer (and also header & footer that doesn't get overlapped by scrollbars). Same thing is being done in `<Launch>` component.